libs/pages/application/src/lib/feature/page-settings-domains-feature/crud-modal-feature/crud-modal-feature.tsx

```diff
@@ -61,6 +65,25 @@ export function CrudModalFeature({
     }
   }
 
+  const { data: links = [] } = useLinks({
+    serviceId: application?.id ?? '',
+    serviceType: getServiceType(application as ApplicationEntity),
+  })
+
+  const refetchLinks = () => {
```

Comment:
You don't need that because `useLinks` already expose a `refetch` method
https://tanstack.com/query/latest/docs/react/reference/useQuery

Reply:
Yes, a the beginning I called `refetch`, but since here I want to invalidateQueries I preferred to do that with queryClient to have the same behavior everywhere, is that good too or we prefer `refetch`? (and the function is here, to have only one call)